cmd/lakectl/cmd/export.go

```diff
@@ -104,12 +101,26 @@ var exportExecuteCmd = &cobra.Command{
 	},
 }
 
+var exportRepairCmd = &cobra.Command{
+	Use:   "repair",
+	Short: "marks failed export as repaired",
```

Comment:
```suggestion
	Short: "mark failed export as repaired",
```
We usually keep documentation of commands in the imperative mode, e.g. see above `Short`s.  Go follows the Google oddity of using "_descriptive verbs in the indicative mood_" (https://google.github.io/styleguide/cppguide.html#Function_Comments), no reason to punish our users...

Reply:
👍